identity-service/src/app.js

```diff
@@ -329,6 +329,19 @@ class App {
     return [listenCountLimiter, listenCountIPLimiter]
   }
 
+  // Create a rate limiter for listens exclusively based on IP
+  _createDailyRateLimitForListenCountIP () {
+    return getRateLimiter({
+      prefix: `listenCountLimiter:::ip:::`,
+      expiry: ONE_HOUR_IN_SECONDS * 24,
+      max: config.get(`rateLimitingListensExclusivelyPerIPPerDay`), // max requests per day
+      keyGenerator: function (req) {
```

Comment:
don't think you have to add this, pretty sure it defaults to ip but dont feel strongly about keeping vs removing

Reply:
we need ip -- the getIP method does a lot of stuff behind the scenes with x-forwarded-for

Reply:
I guess I was thinking it would default to ip because of this https://github.com/AudiusProject/audius-protocol/blob/master/identity-service/src/rateLimiter.js#L121

Reply:
oh you're right. eh, i like the explicitness of it here, but no strong opinion!